specification/maps/data-plane/Search/stable/2023-06-01/search.json

```diff
@@ -0,0 +1,1202 @@
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/UsageType"
+            }
+          }
+        }
+      }
+    },
+    "GeocodingBatchRequestItem": {
+      "description": "Batch Query object",
+      "type": "object",
+      "properties": {
+        "optionalId": {
+          "description": "id of the request which would show in corresponding batchItem",
+          "type": "string"
+        },
+        "top": {
+          "description": "Maximum number of responses that will be returned. Default: 5, minimum: 1 and maximum: 20.",
+          "type": "integer",
+          "format": "int32",
+          "default": 5,
+          "minimum": 1,
+          "maximum": 20
+        },
+        "query": {
+          "description": "A string that contains information about a location, such as an address or landmark name.",
+          "type": "string"
+        },
+        "addressLine": {
+          "description": "The official street line of an address relative to the area, as specified by the locality, or postalCode, properties. Typical use of this element would be to provide a street address or any official address.\n\n**If query is given, should not use this parameter.**",
+          "type": "string"
+        },
+        "countryRegion": {
+          "description": "Restrict the geocoding result to an [ISO 3166-1 Alpha-2 region/country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) that is specified e.g. FR. This will limit the search to the specified region.\n\n**If query is given, should not use this parameter.**",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "bbox": {
+          "description": "A rectangular area on the earth defined as a bounding box object. The sides of the rectangles are defined by longitude and latitude values. For more information, see Location and Area Types. When you specify this parameter, the geographical area is taken into account when computing the results of a location query.\n\nExample: [lon1, lat1, lon2, lat2]",
+          "type": "array",
+          "items": {
+            "type": "number",
+            "format": "double"
+          },
+          "minItems": 4,
+          "maxItems": 4
+        },
+        "view": {
+          "description": "A string that specifies an [ISO 3166-1 Alpha-2 region/country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2). This will alter Geopolitical disputed borders and labels to align with the specified user region.",
```

Comment:
shall we sync this with a description of `View` parameter defined at the top level?